File: easy/challenge_25/main.cpp
// Challenge 25: 
// Easy: Print all odd numbers from 1 to 100
#include <iostream>

int main(int argc, char** argv) {
  for (int i = 1 ; i < 100 ; i++) {
    if (i % 2 != 0) {
      std::cout << i << std::endl;
    }
  }
}

File: easy/challenge_25/Main.java
package code.eval.easy.challenge_25;

// Challenge 25: Odd Numbers
public class Main {

    public static void main(String[] args) {
        for(int i = 1 ; i < 100 ; i+=2) {
            if (i % 2 != 0) {
                System.out.println(i);
            }
        }
    }

}
